rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] The book and person should not be read only, it should be a setter too so I will recommend adding it to the attr_accessor.